lib/main/utils/app_config.dart

```diff
@@ -68,4 +68,6 @@ class AppConfig {
   static String get _platformEnv => dotenv.get('PLATFORM', fallback: 'other');
 
   static bool get isSaasPlatForm => _platformEnv.toLowerCase() == saasPlatform;
+
+  static bool get isWebSocketEchoPingEnabled => dotenv.get('WS_ECHO_PING', fallback: 'false') == 'true';
```

Comment:
add in 
- configuration docs 
- in env example file too

Reply:
In case you missed the adr no. 56

```md
- Echo ping will still be left as an option in `env.file` through `WS_ECHO_PING`
  - Set it to `true` if you want to use Echo ping
  - Set it to `false` or left it as is if you don't want to use Echo ping
```

Reply:
Please should add in configuration docs. User will check it to configure app.

Reply:
Done